MSAL/test/unit/native_auth/public/MSALNativeAuthPublicClientApplicationTest.swift

```diff
@@ -76,6 +76,7 @@ final class MSALNativeAuthPublicClientApplicationTest: XCTestCase {
         let delegate = SignUpPasswordStartDelegateSpy(expectation: exp)
         sut.signUpUsingPassword(username: "", password: "", delegate: delegate)
         wait(for: [exp], timeout: 1)
+        XCTAssertNil(controllerFactoryMock.signUpController.signUpStartRequestParameters?.attributes)
```

Comment:
Do you think these checks are needed for these specific tests? The test names are not describing anything around this. They don't hurt to be there but I think they are a bit overkill. We could have a test where we check that if we don't set the attributes they are not sent in this class but I think it should be separate.

Reply:
Yeah, actually make more sense to have a dedicated test for this, so we can highlight this behaviour. Thanks I'll update this test class